mizar/dp/mizar/operators/endpoints/endpoints_operator.py

```diff
@@ -222,6 +238,8 @@ def create_simple_endpoints(self, interfaces, spec):
             ep.set_vni(spec['vni'])
             ep.set_vpc(spec['vpc'])
             ep.set_net(spec['net'])
+            ep.set_ip(spec['interfaces'][count].get('ip', ''))
+            count += 1
```

Comment:
ditto

Reply:
spec['interfaces'] is a list which includes all the interfaces we need to create, thurs as we loop through each interface, we also need loop through spec['interfaces']. That's why I added count to loop through spec['interfaces']

Reply:
I must be missing something here. How does an ```interface``` maps to ```spec['interfaces'][count]```?

Reply:
spec["interface"] is a list of dictionaries: [create.py](https://github.com/futurewei-cloud/mizar/blob/dev-next/mizar/dp/mizar/workflows/builtins/pods/create.py#L73)
When multiple interfaces are passed in, spec["interface"] = [{"name": "eth0", "ip": "10.10.1.12", "subnet": "net1"}, {"name": "eth1", "ip": "10.10.1.13", "subnet": "net1"}, {"name": "eth2", "ip": "10.10.1.14", "subnet": "net1"}]